electron/index.js

```diff
@@ -116,25 +144,76 @@ function getArgs() {
     if(failed || (argv.help == true)) {
         return false;
     }
+
+    if(argv.rom && !fs.existsSync(argv.rom)) {
+        console.warn('--rom', 'invalid filename', argv.rom);
+        return false;
+    }
+
+    if(argv.map && !fs.existsSync(argv.map)) {
+        console.warn('--map', 'invalid filename', argv.map);
+        return false;
+    }
+
+    if(argv.eeprom && !fs.existsSync(argv.eeprom)) {
+        console.warn('--eeprom', 'invalid filename', argv.eeprom);
+        return false;
+    }
+
+    if(argv.cf && !fs.existsSync(argv.cf)) {
+        console.warn('--cf', 'invalid filename', argv.cf);
+        return false;
+    }
+
     return argv;
 }
 
-function parseArgs(argv) {
+async function parseArgs(argv) {
+    async function setFileInput(wc, selector, files) {
+      try {
+        wc.debugger.attach("1.1");
+
+        const { root } = await wc.debugger.sendCommand("DOM.getDocument", {});
+        const { nodeId } = await wc.debugger.sendCommand("DOM.querySelector", {
+          nodeId: root.nodeId,
+          selector,
+        });
+
+        await wc.debugger.sendCommand("DOM.setFileInputFiles", {
+          nodeId,
+          files,
+        });
+        return true;
+      } catch (err) {
+        return false;
+      } finally {
+        wc.debugger.detach();
+      }
+    }
+
+    let loadAdvanced = false;
+
     if (argv.rom) {
-        mainWindow.webContents.send('rom', fs.readFileSync(argv.rom));
+        loadAdvanced = await setFileInput(mainWindow.webContents, "#file-rom", [path.resolve(argv.rom)]);
     }
     if (argv.map) {
-        mainWindow.webContents.send('map', fs.readFileSync(argv.map));
+        loadAdvanced = await setFileInput(mainWindow.webContents, "#file-map", [path.resolve(argv.map)]);
```

Comment:
Shouldn't it be `loadAdvanced |=` here and below? If any fails, but the remaining succeed, we may still want to load the ones that are loadable